docs/resources/oxide_vpc_firewall_rules.md

```diff
@@ -26,8 +29,8 @@ resource "oxide_vpc_firewall_rules" "example" {
             value = "default"
           }
         ]
-        ports     = ["8123"]
-        protocols = ["ICMP"]
```

Comment:
I changed this example because the name `custom-deny-http` didn't make sense with protocol `ICMP`.